modin/core/execution/dask/common/utils.py

```diff
@@ -54,12 +55,17 @@ def _disable_warnings():
 """,
         )
         num_cpus = CpuCount.get()
+        threads_per_worker = DaskThreadsPerWorker.get()
         memory_limit = Memory.get()
         worker_memory_limit = memory_limit // num_cpus if memory_limit else "auto"
 
         # when the client is initialized, environment variables are inherited
         with set_env(PYTHONWARNINGS="ignore::FutureWarning"):
-            client = Client(n_workers=num_cpus, memory_limit=worker_memory_limit)
+            client = Client(
+                n_workers=num_cpus,
+                threads_per_worker=threads_per_worker,
```

Comment:
What is the default value for this parameter?

Reply:
There is no a default value for this parameter in Dask. Dask computes it based on some logic in depend on available CPUs and n_workers to spawn. I think it makes sense that every distributed engine in Modin by default relies on multiprocessing rather than multithreading+multiprocessing.

Reply:
I saw the following definition:
```python
        if n_workers and threads_per_worker is None:
            # Overcommit threads per worker, rather than undercommit
            threads_per_worker = max(1, int(math.ceil(CPU_COUNT / n_workers)))
```

This doesn't seem so bad. Well, I'm not against this change, but don't we need to split this pull request into 2: adding a new configuration option and changing the default parameter value for Dask?

Reply:
It doesn't make sense to add a config and don't use it anywhere. The change in this PR explicitly says that the default value is 1 and it is used.

Reply:
> It doesn't make sense to add a config and don't use it anywhere. The change in this PR explicitly says that the default value is 1 and it is used.

If you set the default value to `None`, then everything will be as before, but the user will have the opportunity to specify their own values for this variable. Although I don’t insist, it seems to me that this way we will more clearly show that we are changing Dask’s behavior for the users.

Reply:
Makes sense